### ISSUE TITLE
Training hyperparameters for FB15K-237N and CoDeX-S

### ISSUE THREAD

**Issue by luilui97** [zjukg/KoPA#56]
Thanks for your contribution to LLM4KG. Could you please provide training parameters and implement details on FB15k237N on OpenKE? I have tried all the margin settings mentioned in the paper but the accuracy is not as low as reported, using TranE on FB15k237N. Thank you!

**Comment by Zhang-Each** [comment]
Sorry but this problem is not necessarily caused by OpenKE. According to our attempts, many factors affect the final performance of the model, including the configuration of the environment and the version of LLM.

**Comment by luilui97** [comment]
Got it, thanks for your response.
